apps/style/applab/style.scss

```diff
@@ -206,6 +206,7 @@ div#visualizationResizeBar {
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
+  cursor: crosshair;
```

Comment:
Whoops, missed this edit. Its for the next feature addition. I can pull it out if you'd like.